test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/CSharpEndToEndTests.cs

```diff
@@ -320,7 +320,7 @@ public async Task FunctionWithIndexingError_ReturnsError()
         {
             FunctionStatus status = await Fixture.Host.GetFunctionStatusAsync("FunctionIndexingError");
             string error = status.Errors.Single();
-            Assert.Equal("Microsoft.Azure.WebJobs.Host: Error indexing method 'Functions.FunctionIndexingError'. Microsoft.Azure.WebJobs.Extensions.Storage: Storage account 'setting_does_not_exist' is not configured.", error);
+            Assert.Equal("Microsoft.Azure.WebJobs.Host: Error indexing method 'Functions.FunctionIndexingError'. Microsoft.Azure.WebJobs.Extensions.Storage: Storage account connection string 'setting_does_not_exist' does not exist. Make sure that it is a defined App Setting.", error);
```

Comment:
FYI had to update this due to the changes in SDK: https://github.com/Azure/azure-webjobs-sdk/pull/2145